index.js

```diff
@@ -99,7 +99,17 @@ exports.parse = function(str, reviver) {
  */
 exports.replace = function(str, data) {
     return str.replace(/\{\{([^}]+)\}\}/g, function(match, search) {
-        return data[search] ? data[search] : match;
+        if (data.hasOwnProperty(search)) {
```

Comment:
That would exclude `null` as possible variable, similarily to `false`. See the second test case.